transformations/americanize_britishize_english/README.md

```diff
@@ -0,0 +1,25 @@
+# British to American English and vice versa translation
+Author name: Aman Srivastava (amanit0812@gmail.com)
+
+## What type of transformation is this?
+This transformation takes a sentence and converts it from british english to american english and vice-versa. 
+It supports following scenarios:-
+* Sentences with British English words :arrow_right: To American English
+* Sentences with American English words :arrow_right: To British English
+* Sentences with both American English and British English words :arrow_right: Translates American English words to British English words and vice versa.
+
+```
+Examples:
+1. I Love Pastel Colours > I Love Pastel Colors
+2. Carry encyclopedia on vacation > Carry encyclopaedia on holiday
+```
+
```

Comment:
Just as a reminder here. "Robustness Evaluation" section still needs to be added

Reply:
Robustness Evaluation is added

Reply:
Also, if you add the evaluation for the models [here](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/evaluation) this transformation can be added in the leaderboard too.